app/gui/src/dashboard/services/__test__/Backend.test.ts

```diff
@@ -0,0 +1,103 @@
+import { describe, expect, it } from 'vitest'
+import { Rfc3339DateTime } from '../../utilities/dateTime'
+import { AssetType, compareAssets, type AnyAsset } from '../Backend'
+
+describe('Backend', () => {
```

Comment:
lets mix capitalise letters, numbers and special characters in titles to catch more cases